packages/mesos-client/src/rxjs-extensions/keepAlive.js

```diff
@@ -0,0 +1,111 @@
+import { Subscriber, Scheduler } from "rxjs";
```

Comment:
Let's move this operator to the resource service package

Reply:
That actually makes sense!